.gitignore

```diff
@@ -2,3 +2,4 @@ bazel-*
 .bazelrc.user
 .idea/
 .ijwb/
+**/node_modules
```

Comment:
nit: just `node_modules` is sufficient in .gitignore AFAIK

Reply:
Done